product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Understanding quotas in Azure
```

Comment:
I think we should get away from "quotas" here - how about calling this page something like, "Preparing your Azure account for BigAnimal"?

Reply:
Done.